App.js

```diff
@@ -219,20 +221,34 @@ class App extends Component {
       user: null,
       serverError: false,
     }
-    this.onWebSocketMessage = this.onWebSocketMessage.bind(this)
+
+    store.subscribe(() => {
```

Comment:
Nice - we will be able to deal with remote notifications the same way, did you check [this](https://github.com/reactjs/redux/issues/303#issuecomment-125184409) ?

Reply:
Yeah, if you like, I can add a little `observe` utility like the one Dan illustrates in that comment and use that instead of subscribing directly?

Reply:
yeah Dan says it's nicer so let's follow his advice. Maybe in a `notifications` file/folder ?